cddl/concise-swid-tag.cddl

```diff
@@ -60,6 +60,7 @@ $role /= aggregator
 $role /= distributor
 $role /= licensor
 $role /= maintainer
+$role /= module-creator
```

Comment:
We should add this to the IANA section of the draft: the `module-creator` codepoint needs to be added to the Software Tag Entity Role Values Registry.

Reply:
arg, my fault not catching it... this should have gone to `concise-swid-tag-ext.cddl`.  I will fix it in the migration.